airflow/operators/sql_to_gcs.py

```diff
@@ -252,10 +252,36 @@ def _write_local_schema_file(self, cursor):
             contains the BigQuery schema fields in .json format.
         """
         schema = [self.field_to_bigquery(field) for field in cursor.description]
+        tmp_schema_file_handle = NamedTemporaryFile(delete=True)
```

Comment:
Could you also update the block comment, regarding `self.schema` will be honored when provided, otherwise, use  schema read from sq.

Reply:
Thanks @whynick1 the code and comment block has been updated.